index.html

```diff
@@ -64,44 +72,53 @@ <h2 class="head2">Tonic</h2>
                         <div class="link3">javaScript</div>
                     </li>
                 </ul>
-                <button class="see-project1">
+                <button class="see-project1 unique2">
```

Comment:
- [ ] Kindly update the `background-color` of the button in this project card to match the design in Figma 💯 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/206857384-8e372a9f-2087-4a78-bd5a-dacfa05581bb.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/206857400-2f5a49f7-6b4d-4724-b669-d557fa62775e.png)


PS: this images shows the hover effect and not the actual color of the button 👇 
![image](https://user-images.githubusercontent.com/50721479/206857435-0318b5a4-3a63-403a-b692-dfece6f83e16.png)